scss/pages/_home.scss

```diff
@@ -0,0 +1,80 @@
+body {
+    font-family: $font-family-primary;
+    font-size: $body-font-size;
+}
+
+.container {
+    width: 80%;
+    margin: 0 auto;
+    margin-top: 4em;
+    font-size: $body-font-size;
+    line-height: 24px;
+    
+
+    .title {
+        text-transform: capitalize;
+        font-size: $title-font-size;
+        margin-bottom: 0.5em;
+        font-family: $font-family-secondary;
+    }
+
+
+    .section-divide {
+        width: 50px;
+        height: 10px;
+        background-color: $primary-color;
+        margin: 2em 0;
+    }
+
+    .sub-title {
+        color: $accent-color;
+        text-transform: uppercase;
+        font-size: $sub-title-font-size;
+        font-weight: map-get($font-weights, "bold" );
+        letter-spacing: 1px;
+        margin-bottom: 0.4em;
+        
+    }
+
+    .date { 
+        font-size: $body-font-size;
+        font-weight: map-get($font-weights, "regular" );
+    }
+
+    .text {
+        margin-top: 1em;
+        margin-bottom: 3em;
+    }
+
+    #skillsets, #awards, #career-traits, #recent-works {
```

Comment:
Maybe giving them a class would be better? Consistency in your selectors is something nice to practice, not a huge deal though. Just a personal preference.